platform/kobo/koreader.sh

```diff
@@ -82,7 +83,7 @@ fi
 if [ ! -n "${PRODUCT}" ] ; then
 	PRODUCT="$(/bin/kobo_config.sh 2>/dev/null)"
 	[ "${PRODUCT}" != "trilogy" ] && PREFIX="${PRODUCT}-"
-	export PRODUCT
+	export PREFIX PRODUCT
```

Comment:
Then there's no purpose to that line and it should be removed. If it's supposed to be a check then it should be `[  PREFIX="${PRODUCT}-" ]`

As written it says
if product != trilogy
then
prefix=$PRODUCT-